python/generate-invoice/README.md

```diff
@@ -0,0 +1,59 @@
+# 🧾 Generate Invoice
+
+A Python Cloud Function for generating invoice.
+
+_Example input:_
+
+```json
+{
+
+  "payload": {
+    "currency":"EUR",
+    "items":[{"name":"Web development","price":15}],
+    "issuer":"Some\nIssuer",
+    "customer":"Some\nCustomer",
+    "vat":21
+    }
```

Comment:
payload should be a JSON string rather than object.

```suggestion
  "payload": "{\"currency\":\"EUR\",\"items\":[{\"name\":\"Web development\",\"price\":15}],\"issuer\":\"Some\\nIssuer\",\"customer\":\"Some\\nCustomer\",\"vat\":21}"
```